js_modules/dagster-ui/packages/ui-core/src/assets/AssetNodeInstigatorTag.tsx

```diff
@@ -18,26 +19,17 @@ export const AssetNodeInstigatorTag = ({
   assetNode: AssetNodeInstigatorsFragment;
   repoAddress: RepoAddress;
 }) => {
-  const automaterializeSensorsFlagState = useAutomationPolicySensorFlag();
   const {schedules, sensors} = React.useMemo(() => {
-    const schedules = uniqBy(
-      flatMap(assetNode.jobs, (j) => j.schedules),
-      'id',
+    const instigators = assetNode.targetingInstigators;
+    const schedules: ScheduleSwitchFragment[] = instigators.filter(
+      (instigator) => instigator.__typename === 'Schedule',
     );
```

Comment:
I switched this to targetingInstigators but I am having some trouble with the typescript here:

```
Type '({ __typename: "Schedule"; id: string; name: string; cronSchedule: string; executionTimezone: string | null; scheduleState: { __typename: "InstigationState"; id: string; selectorId: string; status: InstigationStatus; }; } | { ...; })[]' is not assignable to type 'ScheduleSwitchFragment[]'.
  Type '{ __typename: "Schedule"; id: string; name: string; cronSchedule: string; executionTimezone: string | null; scheduleState: { __typename: "InstigationState"; id: string; selectorId: string; status: InstigationStatus; }; } | { ...; }' is not assignable to type 'ScheduleSwitchFragment'.
    Type '{ __typename: "Sensor"; id: string; jobOriginId: string; name: string; sensorState: { __typename: "InstigationState"; id: string; selectorId: string; status: InstigationStatus; }; }' is missing the following properties from type 'ScheduleSwitchFragment': cronSchedule, executionTimezone, scheduleState
```

It seems like it's not smart enough to realize that i've set up the fragment so that if the type is a Sensor, it uses the correct fragment - @salazarm or @hellendag do you happen to know how I can make it realize that, or should I set up my query differently?

(I could also make a separate targetingSchedules field I suppose)